nexus/proposals/tests/test_models.py

```diff
@@ -0,0 +1,45 @@
+# Standard Stuff
+
+# Third Party Stuff
+from django.test import TestCase
+from tests import factories as f
+
+# nexus Stuff
+from nexus.proposals.models import Proposal, ProposalKind
+
+
+class TestProposalModel(TestCase):
+
+    def setUp(self):
+        self.user = f.create_user(email='test@example.com', password='test')
+        self.proposalkind = f.create_proposal_kind(kind='talk')
+
+    def test_create_proposal(self):
+        proposal = {
+            'speaker': self.user,
+            'title': 'Test Proposal',
+            'kind': self.proposalkind,
+            'level': 'beginner',
+            'duration': '00:30:00',
+            'abstract': 'Proposal abstract',
+            'description': 'Proposal description',
+        }
+        proposal = Proposal.objects.create(**proposal)
+        assert proposal.id
+        assert proposal.speaker == self.user
+        assert proposal.title == 'Test Proposal'
+        assert proposal.kind == self.proposalkind
+        assert str(proposal.level) == 'beginner'
+        assert proposal.duration == '00:30:00'
+        assert proposal.abstract == 'Proposal abstract'
+        assert proposal.description == 'Proposal description'
+        assert proposal.accepted_at is None
+        assert proposal.status == 'submitted'
```

Comment:
Pick these from the extended choices, don't use the raw value that is there currently.